README.md

```diff
@@ -27,32 +27,57 @@ Supported SQL:
 - PostgreSQL
 - Trino SQL
 
+**Supported helper methods**
+
+| SQL Type    | SQL Split | Auto-Complete |
+| ----------- | -------- | -------- |
+| Generic SQL | WIP      | WIP      |
+| Flink SQL   | ✅        | ✅        |
+| Spark SQL   | ✅        | ✅        |
+| Hive SQL    | ✅        | ✅        |
+| PL/SQL      | WIP      | WIP      |
+| Postgre SQL | WIP      | WIP      |
+| Trino SQL   | WIP      | WIP      |
+
 >Tips: This project is the default for Javascript language, also you can try to compile it to other languages if you need.
 
+<br/>
+
 ## Integrating SQL Parser with Monaco Editor
 
 We have provided a [monaco-sql-languages](https://github.com/DTStack/monaco-sql-languages) package, you can integrate with `monaco-editor`
 easily.
 
+<br/>
+
 ## Installation
 
 ```bash
-// use npm
+# use npm
 npm i dt-sql-parser --save
 
-// use yarn
+# use yarn
 yarn add dt-sql-parser
 ```
 
+<br/>
+
 ## Usage
+Before you get started, you need to understand the basics of how to use it. `dt-sql-parser` provides SQL parser classes for different types of supported SQL:
+```javascript
+import { GenericSQL, FlinkSQL, SparkSQL, HiveSQL, PLSQL, PostgresSQL, TrinoSQL } from 'dt-sql-parser';
+```
 
-### Syntax Validation
+Before using syntax validation, autocompletion, and other method, you need to instantiate the Parser of the corresponding SQL type, taking `GenericSQL` as an example:
+```javascript
+const parser = new GenericSQL();
+```
 
-First, we need to import the **Parser** object from `dt-sql-parser`, the different language needs
-different Parser, so if you need to handle the **Flink SQL**, you can import the **FlinkSQL Parser**.
+The usage examples below will use `GenericSQL`, and Parser for other SQL types will be used in the same way as `GenericSQL`.
```

Comment:
The following usage examples will utilize the `GenericSQL` language, and the Parser for other SQL types will be employed in a similar manner as `GenericSQL`.